backstory/src/main/webapp/perspective-demo.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Perspective</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato" />
+    <script src="perspective.js"></script>
```

Comment:
Does this file need to get checked in first (I don't see it in this PR, and also don't see it in the repo yet)?

Reply:
@ykjchen This file is added in next commit. I cherrypicked this commit from other perspective work, which is why it's here. It will be added in the next commit. I'll just remove it for the moment.

Reply:
Done